docs/installation/migrate.rst

```diff
@@ -41,50 +62,77 @@ The detailed procedure is as follows:
     
 #.  Start Replicator so that it can perform the one-time schema migration. 
 
+    .. code:: bash
+    
+        TBD example "start replicator" command
+
 #.  Stop all producers.
 
+    .. code:: bash
+
+        TBD example "stop producers" command
+
 #.  Wait until the replication lag is 0.
 
+    .. code:: bash
+
+        TBD example output 
+
 #.  Stop Replicator.
 
+    .. code:: bash
+
+        TBD example "stop producers" command
+
 #.  Enable mode changes in the source Schema Registry by adding the following to the
-    configuration and restarting.  (Note: modes are only supported in version 5.2 of
-    Schema Registry.  Steps 7 and 8 are only precautionary and are not strictly
-    necessary.  If using version 5.1 of Schema Registry or earlier, skips 7 and 8
-    can be skipped, as long as all producers have indeed been stopped so that no
-    further schemas are registered in the source Schema Registry.)
+    configuration and restarting.  
     
     
     .. code:: bash
```

Comment:
I try not to use "bash" for stuff that's not an actual shell command -- for settings stuff, I use the plain `::` syntax, `.. code:: text`.